marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                "x": 73,
+                "y": 54
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "These information are available for all the databases included in `resource_filters`, so the most important ones can be monitored here",
+                "font_size": "14",
+                "show_tick": true,
+                "text_align": "center",
+                "tick_edge": "right",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 3596562826502024,
+            "layout": {
+                "height": 18,
+                "width": 15,
+                "x": 112,
+                "y": 5
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.databases.total_rate{$database_name} by {database_name}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Sum of the processing rate factors by $database_name",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 7873483361703650,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 128,
+                "y": 31
+            }
+        },
+        {
+            "definition": {
+                "autoscale": true,
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "conditional_formats": [
+                            {
+                                "comparator": ">",
+                                "palette": "white_on_green",
+                                "value": 0
+                            },
+                            {
+                                "comparator": "<=",
+                                "palette": "white_on_yellow",
+                                "value": 0
+                            }
+                        ],
+                        "q": "sum:marklogic.databases.backup_count{$database_name}"
+                    }
+                ],
+                "time": {},
+                "title": "Number of backup for $database_name",
```

Comment:
- The title is cut off in the dashboard
- I think this is missing a word after "backup"